Packs/DarktraceEventCollector/Integrations/DarktraceEventCollector/DarktraceEventCollector_description.md

```diff
@@ -0,0 +1,4 @@
+## Darktrace
```

Comment:
```suggestion
**Important:** Despite being part of a partner-supported pack, this integration is supported by Palo Alto Networks.

## Configure Darktrace Event Collector
```
@meital-aizen @ShahafBenYakir objections?

Reply:
according to @meital-aizen,
```
**Important:** 
This integration is supported by Palo Alto Networks
```